aptos-move/move-examples/upgrade_and_govern/v1_1_0/scripts/set_and_transfer.move

```diff
@@ -0,0 +1,18 @@
+// :!:>script
+script {
+    use upgrade_and_govern::parameters;
```

Comment:
I would avoid placing multiple scripts in the same package and entirely from modules, reason being, they don't keep their names as they get compiled, so you don't know which script is which.

Reply:
> they don't keep their names as they get compiled, so you don't know which script is which

@davidiw I'm a bit confused by this guidance, as the upgrade package has two scripts, `set_and_transfer.move` and `set_only.move`, which respectively compile into `set_and_transfer.mv` and `set_only.mv` as expected.

I suspect the hypothetical name loss you refer to is a result of using `main` as the main function name for every script in a Move package:

```move
script {
    fun main( // <- main function name
    //  ^ this PR uses either `set_only` or `set_and_transfer`, but
    //    name loss would result if both had `main`
        ...
    ) {
        ...
    }
}
```

This is analogous to using the same module definition block across multiple module `.move` files

```move
module named_address::a_repeated_module_name {
    //                ^ this compiles into `a_repeated_module_name.mv`
    //                  regardless of the `.move` filename
    ...
}
```

What are your thoughts?